docs/daemon/kwild.mdx

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_position: 3
+sidebar_label: "Using Kwild"
+id: daemon-kwild
+title: Running a Kwil Node
+description: How to deploy a Kwil node using kwild
+slug: /daemon/kwild
+---
+
+The `kwild` command starts the blockchain and the database services of the Kwil node. This guide provides step-by-step instructions to launch a Kwil node.
+
+## Root Directory
+
+This is the default directory to store all the configuration and data such as blockchain data, SQL database data, signed messages, etc, unless specified otherwise.
+
+```
+home_dir/
+   |- config.toml
+   |- private_key         
+   |- abci/                     # chain root directory, stores blockchain data & config
+   |   |- config/
+   |   |   |- genesis.json   
+   |   |- data/ 
+```
+
+The default root directory is `~/.kwild`. You can override this configuration by setting the `KWILD_ROOT_DIR` environment variable or using the `--root_dir` command line flag.
+
+## Quickstart
+
+Quickstart mode can be used to swiftly deploy a node without delving into the configuration details.  To run in this mode, use the `kwild` command with the `--autogen` flag.
+
+```bash
+kwild --autogen
+```
+This command will auto-generate the necessary configuration in the root directory and starts the node using this configuration. This also create a new private chain with the initialized node as its sole member and validator. 
+
+It's crucial to note, however, that the node operates under a default configuration in the quickstart mode. For production deployments, it's highly recommended to tailor the configuration to your specific needs, as discussed in subsequent sections. 
+
+## Configuration
+
+Running a Kwil node requires `genesis.json`, `config.toml` and `private_key` files. All these configuration files must be present under the root directory. This section describes in detail on how to initialize the root directory with these configuration files. 
+
+### Private Keys
+
+Kwil nodes use e25519 keys for validator related operations. Refer to the [key management guide](./../admin/keys#generation) for generating these node keys. The node's private key file must reside at location `root_dir/private_key`, else configure the location using `app.private_key_path` in `config.toml`.
+
+### Genesis File (genesis.json)
+
+To deploy a new Kwil network, copy the sample [genesis.json](./config/genesis#complete-genesis-configuration) into `root_dir/abci/config/genesis.json`. Manually update the `chain_id`, `initial_height`, `app_hash` configurations in the `genesis.json` file.
+
+To start the node as a validator, the node's base64 public key and address must be listed under the `validators` in the genesis file as shown below. Refer to the [key management guide](./../admin/keys#key-information) to view the public key, address and node id corresponding to the node's private key.
```

Comment:
We changed this to hex, right?

Reply:
Also, we use the term "node ID" and "node id" throughout the docs.  We should standardize this; I think "node ID" is preferable, but don't care too much.

Reply:
Yeah, Jon's making Hex change. Thought to update this once his changes are in. address config might be removed too from genesis

Reply:
Gotcha.  Ok, we can include those in the next PR